src/DssSpellManager.t.sol

```diff
@@ -0,0 +1,126 @@
+pragma solidity 0.6.12;
+
+import {DssSpellTest, TinlakeManagerLike} from "./DssSpell.t.sol";
+import "dss-interfaces/Interfaces.sol";
+
+interface EpochCoordinatorLike {
+    function closeEpoch() external;
+    function currentEpoch() external returns(uint);
+}
+
+interface Hevm {
+    function warp(uint) external;
+    function store(address,bytes32,bytes32) external;
+    function load(address,bytes32) external returns (bytes32);
+}
+
+interface Root {
+    function relyContract(address, address) external;
+}
+
+interface MemberList {
+    function updateMember(address, uint) external;
+}
+
+interface AssessorLike {
+    function calcSeniorTokenPrice() external returns (uint);
+}
+
+contract DssSpellManager is DssSpellTest {
+    address self;
+
+    DSTokenAbstract  public drop;
+    TinlakeManagerLike dropMgr;
+
+    // Tinlake
+    Root constant root = Root(0x53b2d22d07E069a3b132BfeaaD275b10273d381E);
+    MemberList constant memberlist = MemberList(0x5B5CFD6E45F1407ABCb4BFD9947aBea1EA6649dA);
+    EpochCoordinatorLike constant coordinator = EpochCoordinatorLike(0xFE860d06fF2a3A485922A6a029DFc1CD8A335288);
+    address constant seniorOperator_ = 0x230f2E19D6c2Dc0c441c2150D4dD9d67B563A60C;
+    address constant seniorTranche_ = 0xfB30B47c47E2fAB74ca5b0c1561C2909b280c4E5;
+    address constant assessor_ = 0xdA0bA5Dd06C8BaeC53Fa8ae25Ad4f19088D6375b;
+
+    function managerInit() public {
+        self = address(this);
+
+        hevm.store(
+            mgr_, keccak256(abi.encode(address(this), uint(0))), bytes32(uint(1))
+        );
+        assertEq(mgr.wards(self), 1);
+
+        // setup manager dependencies
+        mgr.file("urn", address(rwaurn));
+        mgr.file("liq", address(oracle));
+        mgr.file("owner", self);
+
+        super.setUp();
+
+        dropMgr = TinlakeManagerLike(address(mgr));
+        drop = DSTokenAbstract(address(dropMgr.gem()));
+
+        // welcome to hevm KYC
+        hevm.store(address(root), keccak256(abi.encode(address(this), uint(0))), bytes32(uint(1)));
+
+        root.relyContract(address(memberlist), address(this));
+        memberlist.updateMember(address(this), uint(-1));
+        memberlist.updateMember(address(dropMgr), uint(-1));
+
+        // set this contract as owner of dropMgr // override slot 13
+        // check what's inside slot 1 with: bytes32 slot = hevm.load(address(dropMgr), bytes32(uint(13)));
+        hevm.store(address(dropMgr), bytes32(uint(13)), bytes32(uint256(address(this))));
```

Comment:
this line is not required anymore. Artefact from the tests of the first mgr version

Reply:
removed

Reply:
Removed!